EngineLayer/ClassicSearch/ClassicSearchEngine.cs

```diff
@@ -104,7 +104,7 @@ protected override MetaMorpheusEngineResults RunSpecific()
                         var ListOfModifiedPeptides = peptide.GetPeptidesWithSetModifications(variableModifications, maximumVariableModificationIsoforms, max_mods_for_peptide).ToList();
                         foreach (var yyy in ListOfModifiedPeptides)
                         {
-                            var correspondingCompactPeptide = yyy.CompactPeptide;
+                            var correspondingCompactPeptide = yyy.CompactPeptide();
```

Comment:
Let's pass lp here, what do you think? You might get some minor space savings, but, more importantly you could get rid of the parameter-less constructor of compactpeptide.